Makefile

```
@@ -5,50 +5,72 @@ include $(abspath $(PROJECT_DIR)/build/automation/init.mk)
 # Development workflow targets
 
 setup: # Set up project
-	make project-config
-	make python-virtualenv
-	pip install -r application/requirements.txt
+	make docker-build NAME=serverless
+	make serverless-requirements
+	make python-requirements
 
 build-dev: # Build dev requirements
 	make docker-build NAME=serverless
 	make serverless-requirements
 	make python-requirements
 
 build: # Build lambdas
-	make event-sender-build
-	make event-receiver-build
+	make event-sender-build AWS_ACCOUNT_ID_MGMT=$(AWS_ACCOUNT_ID_NONPROD)
```

I missed this one in my review but can you add this
```suggestion
  make event-sender-build AWS_ACCOUNT_ID_MGMT=$(AWS_ACCOUNT_ID_NONPROD)
  make event-receiver-build AWS_ACCOUNT_ID_MGMT=$(AWS_ACCOUNT_ID_NONPROD)
```